build.gradle.kts

```diff
@@ -148,9 +149,9 @@ dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter-params:$jupiterVersion")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$jupiterVersion")
 
-    testImplementation("io.github.bonigarcia:selenium-jupiter:3.3.0")
+    testImplementation("io.github.bonigarcia:selenium-jupiter:3.4.0")
```

Comment:
There is a 5.x but it breaks a bunch of tests and I didn’t have time to dig into it.